docs/manifest.json

```diff
@@ -33,16 +43,11 @@
           "path": "./CODE_OF_CONDUCT.md"
         },
         {
-          "title": "Contributors",
-          "description": "See a contributors to Coder.",
-          "path": "./CONTRIBUTORS.md"
+          "title": "Security policy",
+          "description": "Learn how to report vulnerabilities in Coder.",
```

Comment:
```suggestion
          "description": "Our policy for security vulnerabilities in Coder.",
```

Reply:
Eh, this isn't really a policy -- maybe we just call it "Security" and leave it as "How to report vulnerabilities in Coder"

Reply:
Fixed to Katie's point